test/scenarios.py

```diff
@@ -4057,6 +4057,187 @@
             ),
         },
     ),
+    Scenario(
```

Comment:
Could you add a test that sends `/tmp/nested`  so to test the multiple levels of directories?

Reply:
Yeah, so there is one more thing. I moved the dir creation code later during the transfer. It was done upload calling download. But I moved it when the downloaded temporary is placed into the destination.
So basically this test case tests nothing related to the directory resume TBH.
I must have missed it somehow

Reply:
I've changed the test case to use `/tmp/nested`  dir